docs/development/extensions.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Extensions
+description: ✨ Information on developing an Extension for Spicetify.
+---
+
+## How do they work?
+TBA
+
+## What are the guidelines?
+TBA
+
+## Limitations
+TBA
```

Comment:
I'd remove this file and simply create an issue on the repo so that we can create it later on. I do not feel comfortable pushing this to main.